DoePaAdminViewModel/Services/DoePaAdminService.cs

```diff
@@ -108,6 +110,35 @@ public async Task<Anstellungsdetail> CreateAnstellungsdetailAsync(CancellationTo
             return await AddDataToDbSetAsync(DBContext.Anstellungsdetails, cancellationToken);
         }
 
+        public async Task<IEnumerable<EmployeeInvoicedHours>> GetEmployeeInvoicedHoursAsync(string email, DateTime from, DateTime to, CancellationToken cancellationToken = default)
+        {
+            var result = DBContext
+                .Ausgangsrechnungspositionen
+                        .Include(arp => arp.ZugehoerigeRechnung)
+                        .Include(arp => arp.ZugehoerigeKostenstelle)
+                        .Include(arp => arp.ZugehoerigeAuftragsposition)
+                            .ThenInclude(zap => zap.Auftrag)
+                                .ThenInclude(a => a.ZugehoerigesProjekt)
+                                    .ThenInclude(p => p.Rechnungsempfaenger)
+                                        .ThenInclude(r => r.ZugehoerigerKunde)
+                        .Include(arp => arp.ZugehoerigeAuftragsposition.Auftrag.VerantwortlicherMitarbeiter)
+                        .Include(arp => arp.ZugehoerigeAbrechnungseinheit)
+                    .Where(arp => arp.ZugehoerigeAuftragsposition.Auftrag.VerantwortlicherMitarbeiter.Email == email)
+                    .Where(arp => arp.ZugehoerigeAbrechnungseinheit.Name.Equals("Stunden"))
+                    .Where(arp => arp.LeistungszeitraumVon <= to)
+                    .Where(arp => arp.LeistungszeitraumBis >= to)
+                    .ToList()
+                    .GroupBy(arp => arp.ZugehoerigeAuftragsposition.Auftrag.ZugehoerigesProjekt)
+                    .Select(project => new EmployeeInvoicedHours {
+                        Month = string.Format("{0}-{1}", to.Year, to.Month),
+                        Project = project.Key.Projektname,
+                        Customer = project.Key.Rechnungsempfaenger.ZugehoerigerKunde.Kundenname,
+                        HoursCount = (double)project.Sum(p => p.Stueckzahl),
```

Comment:
Is there a reason for HoursCount to be a double? Can't we just change it to decimal, so that it behaves like all similar properties (which are all decimals)?

Reply:
I'll change it